### PR TITLE
Add section for Workload namespace configuration

### PR DESCRIPTION
**Pull request by sclevine**

### CODE REVIEW

**Comment by jncd** [comment]
Thanks! @sclevine

File: install.md [hunk 1]
```diff
@@ -1423,3 +1423,74 @@ Use the following procedure to verify that the packages are installed.
...
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: registry-credentials
+  annotations:
+    secretgen.carvel.dev/image-pull-secret: ""
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: e30K
```
Comment (evankanderson) [inline install.md:1452]: These seem like they are redundant (they will be filled with the same credentials).

File: install.md [hunk 2]
```diff
@@ -1423,3 +1423,74 @@ Use the following procedure to verify that the packages are installed.
...
+kind: Role
+metadata:
+  name: kapp-permissions
+  annotations:
+    kapp.k14s.io/change-group: "role"
+rules:
+  - apiGroups:
+      - serving.knative.dev
+    resources: ['services']
+    verbs: ['*']
```
Comment (evankanderson) [inline install.md:1476]: Worth a note that this might also need to cover resources like service bindings if in use?

**Review by evankanderson — COMMENTED** [review]
Awesome, thanks for this!

I wrote up something similar in #183 -- do you think we should have both, or link to one from the other?

**Comment by a-peek4** [comment]
@evankanderson @sclevine Please take a look at the comments and make a new PR if there are any doc changes to be made. Thank you!

**Pull request merged**